sign-up-task/src/Utils/validators.js

```diff
@@ -37,5 +39,32 @@ export const fieldValidator = (fieldName, value = '', passwordValue = '') => {
 
         default:
     }
+}
 
+export const signUpFormValidator = async (newUser, email) => {
+    let users = await signUpAPI.getUsers();
+    let isUser = users.some(user => user.email === email)
+    isUser && signUpAPI.setUser(newUser)
+    return isUser;
 }
+
+export const signInFormValidator = async (input, email, password) => {
```

Comment:
why you need input here?

Reply:
garbage is left after refactoring((
mu fault